### DISCUSSION TITLE
[Feature] Set user agent of mobile app to "Immich" and the version number

### DISCUSSION THREAD

**Discussion by jrcichra** [immich-app/immich#12467] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Looking at Cloudflare analytics for my Immich instance, I see Immich mobile traffic has a user agent of `Dart/3.5 (dart:io)`. I know that Dart is the programming language commonly used by mobile apps but I think it would be nice if the user agent was more descriptive that the requests were coming from the Immich mobile app, perhaps with the version number of the mobile app.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
#18478
